app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -36,3 +36,10 @@
         = multiple_relationship_link(@record, :container)
         = multiple_relationship_link(@record, :container_image)
         = single_relationship_link(@record, :container_node)
+        - if @record.number_of(:persistent_volumes).zero?
```

Comment:
This is wrong, you need to use one of the `*relationship_link` functions.

Any reason you can't do that?